### PR TITLE
Import Vendors as Merchants

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: apps/mappings/imports/modules/merchants.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        Trigger import for Merchant module
+        """
+        self.check_import_log_and_start_import()
+
+    # remove the is_auto_sync_status_allowed parameter
+    def construct_fyle_payload(
+        self,
+        paginated_destination_attributes: List[DestinationAttribute],
+        existing_fyle_attributes_map: object,
+        is_auto_sync_status_allowed: bool
```
Comment (NileshPant1999) [inline apps/mappings/imports/modules/merchants.py:35]: this is not used anywhere

Comment (NileshPant1999) [reply apps/mappings/imports/modules/merchants.py:35]: please remove

File: tests/test_mappings/test_imports/test_modules/test_merchants.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    # create new case
+    paginated_destination_attributes = DestinationAttribute.objects.filter(workspace_id=1, attribute_type='MERCHANT')
+
+    existing_fyle_attributes_map = {}
+    is_auto_sync_status_allowed = merchant.get_auto_sync_permission()
+
+    fyle_payload = merchant.construct_fyle_payload(
+        paginated_destination_attributes,
+        existing_fyle_attributes_map,
+        is_auto_sync_status_allowed
```
Comment (NileshPant1999) [inline tests/test_mappings/test_imports/test_modules/test_merchants.py:17]: please to remove

File: apps/mappings/imports/modules/merchants.py (hunk 1 above)
Comment (ruuushhh) [reply apps/mappings/imports/modules/merchants.py:35]: image
Then Should we use if condition here?

Comment (NileshPant1999) [reply apps/mappings/imports/modules/merchants.py:35]: ohh okay got it please ignore my comment then

**Review by NileshPant1999 — APPROVED** [review]

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**